app/controllers/reviews_controller.rb

```diff
@@ -6,22 +6,34 @@ def new
   end
 
   def create
-    shelter = Shelter.find(params[:shelter_id])
-    review = shelter.reviews.new(review_params)
+    @shelter = Shelter.find(params[:shelter_id])
```

Comment:
Why did we make this local variable an instance variable?

Reply:
Looks good.  Per our conversation, it might be a good question to ask the instructors.